src/app/helpers/capitalize.js

```diff
@@ -0,0 +1 @@
+export default value => value.charAt(0).toUpperCase() + value.slice(1);
```

Comment:
a case, what if the value will be all in capital letters, like `'HULK'`
How would you do capitalize on it? )
It's not necessary, because maybe we even don't want such a result. 
Just to think about.
And the comment to code, I would recommend to create a named export and avoid using `default` anywhere